### ISSUE TITLE
form-group labels overlap other objects on the page

### ISSUE THREAD

**Issue by hristovh** [vmware/clarity#941]
**Select one ...**  (check one with "x")
```
[x] bug
[ ] feature request
[ ] enhancement
```

### Expected behavior
The div.form-group element should resize according to the nested input and label elements

### Actual behavior
The div.form-group element resizes according to the nested input only

### Reproduction of behavior

* Include a link to the reproduction scenario you created by forking one of the Clarity Plunker Templates:

https://plnkr.co/edit/ilCAllBWbTxzn16MgAkO?p=preview
### Environment details

* **Angular version:** None

* **Clarity version:** Latest - 0.9.4

* **OS and version:** OS X El Capitan

* **Browser:** [Chrome XX | Firefox XX | IE XX | Safari XX]

**Comment by adityarb88** [comment]
Hey @hristovh: We already have this issue raised (see here: https://github.com/vmware/clarity/issues/626) and we are looking into this. The fix that we have identified might be a breaking change and we are evaluating the impact of it. For now the best way to deal with forms which might have wrapping labels is to use forms inside of grids (https://vmware.github.io/clarity/documentation/forms - Refer to the "Forms in Grid" sections). Closing this issue as a duplicate.

**Comment by hippee-lee** [comment]
Hey there @hristovh I took a look at your plunker.
wrt your markup structure: `.form-group` and `.alert .alert-warning` are different in that `form-group` class is for layout and `alert*` classes are for presentation.

Take a look here where I modified your markup: https://plnkr.co/edit/tI05y6ueqEKA06fK2vL7?p=preview

While it doesn't fix the underlying issues with forms it might help you mitigate it until we address that.
